samples/04_gis_analysts_data_scientists/Training a Deep Learning model, for using it in 3D basemap Solutions..ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
<p>Once the model has been saved, it is then ready to be implemented in the deep learning tools provided with the solution. As the deep learning tools are object-specific, it is important to use the appropriate tool. For example, if a user trains a model specifically for tree points, then the 'classify tree points using deep learning' tool should be used. The provided object-specific deep learning tools also expect the model's classcode to be specific. For example, the 'classify tree points using deep learning' tool expects the user-trained model's classcode for tree points to be '5'. The user-trained model's classcode can be remapped to some other classcode when writing in the <code>.las</code> files.</p><p> </p><p>This notebook has summarized the workflow for the training of a deep learning model for point cloud classification, intended to be used when the bundled models in the 3D Basemap solution are not utilized.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
corrected accordingly, Thanks.